content/en/blog/2024/2024-03-29-first-vote-results.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "First vote results"
+linkTitle: First vote results
+date: 2024-03-29
+author: dma ([@dma](https://hachyderm.io/@dma))
+description: Results of the first Nivenly member vote
+---
+
+Earlier this month, we sent out the first Nivenly Member vote covering Pachli, 
+Haidra, and general feedback.  As per our bylaws, we need 50% of the membership
+to participate in a vote for quorum, and 50% of total voting members to be in
+consensus for any result to be binding.  As we have 74 members, this means 37
+needed to vote and 18 needed to agree to any individual item.
+
+Unfortunately, only 14 responses were registered in total meaning that the 
+results of the vote are not binding.  However, they do still provide us with a
+direction from the Nivenly members, and that will help inform the decisions the
+Board now makes around these issues.
+
+Thank you very much to everyone that participated in discussions around these 
+projects, and special thanks to the members that voted.
+
+The rest of this post will cover each question in turn with the results, and 
+how we are interpreting the results.
+
+## New project proposals
+
+### Haidra: member project or incubated project?
+
+This was a question we anticipated would be fairly split and indeed the results
+are, with exactly 50% voting "NO" to Haidra being either a regular or incubated
+project, and 50% voting "YES".
+
+7% voted that Haidra should reapply in a year with any concerns mitigated.
+
+The Board will take this advice from the membership and decide what to do with
+Haidra in our next meeting.
+
+### Pachli: member project or incubated project?
+
+This was much less contentious, with 78.6% voting "YES" to Pachli being adopted 
+as a project.  50% voted that it could be a member project directly.
```

Comment:
Similar for phrasing: 

* Title indicates there was no "no" option
* "This was much less contentious - 78.6% voted yes for Pachli to be a member project and of those N% voted for it to be a member project and M% voted for it to be an incubated project."